packages/data-store/src/migrations/generic/8-CreateContacts.ts

```diff
@@ -0,0 +1,66 @@
+import { DatabaseType, MigrationInterface, QueryRunner } from 'typeorm'
```

Comment:
did niels not mention we should not create new migration files?

"But before we do that I would like someone to go over all existing migrations and code as we are combining different naming strategies for tables etc"

Reply:
I interpreted that as a future TODO. We already have multiple migration files. I'm happy to flatten them, but "someone has to go over all existing migrations and code" when we have time.
For now Maas created 8-CreateContacts.ts, I created 9-CreateContacts.ts
Separate files does have the advantage of better separation of concern => better readability rather than one giant worm of SQL lines.

Reply:
please check with niels

Reply:
See Jira remark